intuita-webview/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
can it use the global `.gitignore`?

Reply:
i think i can just remove this nested .gitignore